@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -190,11 +190,11 @@
                   </tr>
                 </thead>
                 <tbody>
-                  <tr v-for="(networkInterface, index) in vmState.networkInterfaces" :key="index">
+                  <tr v-for="(networkInterface, index) in visibleNetworkInterfaces" :key="index">
```

Comment:
```suggestion
   <tr v-for="(networkInterface, index) in vmState.networkInterfaces" :key="index">
```
I think you can keep the old version